addons/project/models/project.py

```diff
@@ -2128,7 +2064,11 @@ def write(self, vals):
         recurrence_update = vals.pop('recurrence_update', 'this')
         if recurrence_update == 'future':
             for task in self:
-                recurrent_task = task.ancestor_id if task.ancestor_id.recurring_task else task
+                #ABGH: TO REMOVE Later BY AUON
```

Comment:
I think @auon-odoo have already removed it in his task, so no need to add this comment. 🙂